integr8ly/integreatly-operator
# MGDAPI-1857:bump cro to v0.27.0, apply the signature changed for croUtil.ReconcilePosgres for postgres cr flag applyImmediately

# Description


add the applyImmediatly flag to the posgres cr's in

- amqonline
- codeready
- fuse
- ups
- threescale
- rhsso
- user-sso


## Verification
May need to install rhmi and rhoam
- install RHOAM/RHMI with `useClusterStorage:false`
- check that the version of CRO is set to v0.27.0 in the rhmi CR
- check the postgres CR for `applyImmediately: true` for the following products
### rhmi install includes rhoam only install
  - amqonline
  - codeready
  - fuse
  - ups
 ### rhoam only install
  - threescale 
  - rhsso
  - user-sso

- confirm in the aws console that the rds has upgraded the engine version 
> **NOTE:** if you don't have aws console access you can use the aws cli to check the engine version
```bash
# This returns the instance Identifier
oc get postgres example-postgres -o json | jq '.metadata.annotations.resourceIdentifier'
"mwcollabbyocf88tpcloudresourceopera-tquz"
# Check the status to see , should go to `upgrading` after re-running the operator and then `available` once its finished upgrading  
$ aws rds describe-db-instances --db-instance-identifier=mwcollabbyocf88tpcloudresourceopera-tquz --region=eu-west-2 |  jq '.DBInstances[0].DBInstanceStatus'
# This returns the info on the rds instance engine version
$ aws rds describe-db-instances --db-instance-identifier=mwcollabbyocf88tpcloudresourceopera-tquz --region=eu-west-2 |  jq '.DBInstances[0].EngineVersion'
```

## Checklist
